src/math/Triangle.js

```diff
@@ -106,6 +111,25 @@ class Triangle {
 
 	}
 
+	static getAttributeInterpolationFromBarycoord( attr, i1, i2, i3, barycoord, target ) {
```

Comment:
The change is okay for me 👍 

Would you still add a documentation for `getAttributeInterpolationFromBarycoord()`?

Reply:
Done!

Reply:
How about simply `getInterpolatedAttribute()`.

It is more than just `barycoord` that you are passing in.

Reply:
Just renamed it! I think the simpler name is better, as well.